playbooks/ansible-tox-molecule/pre.yaml

```diff
@@ -1,5 +1,18 @@
 ---
 - hosts: all
   tasks:
+
     - name: Reset SSH connection for new group
       meta: reset_connection
+
+    - name: Workaround for yum --nobest with docker-ce
+      when:
+        - ansible_distribution == "CentOS"
+        - ansible_distribution_major_version == "8"
+      become: true
+      yum:
+        name: https://download.docker.com/linux/centos/7/x86_64/stable/Packages/containerd.io-1.2.6-3.3.el7.x86_64.rpm
```

Comment:
I'd like to see us link to some sort of upstream issue around this, as we likely should fix this in the install-docker role.

Reply:
follow up on https://review.opendev.org/#/c/703053/